src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -142,6 +142,8 @@ class SaveInProgressIntro extends React.Component {
     } else if (renderSignInMessage) {
       alert = renderSignInMessage(prefillEnabled);
     } else if (prefillEnabled && !verifyRequiredPrefill) {
+      // If you pass in a headingLevel prop it will change what is rendered by the changeSaveInProgressHeadingLevel helper
+      const H = `${this.props.headingLevel}`;
```

Comment:
Can you rework this so that `h` is at the beginning of the template string? That way the `headlingLevel` prop can just be an int. The way it is now, someone could do `headingLevel="div"` and we end up with a `<div>`, which is not what we want.

With this change, the propTypes would need to be changed from `string` to `number` as well.

Reply:
@bkjohnson Yep, I totally agree. Refactored this.